llvm/include/llvm/ADT/SmallPtrSet.h

```diff
@@ -352,7 +354,7 @@ class SmallPtrSetImpl : public SmallPtrSetImplBase {
   }
 
   /// erase - If the set contains the specified pointer, remove it and return
-  /// true, otherwise return false.
+  /// true, otherwise return false. Invalidates iterators if it returns true.
```

Comment:
nit: In the first line, the documentation doesn't really explain what this function does. Also, I'd remove the `erase - ` prefix -- not sure where this came from.